src/main/kotlin/ar/edu/unq/readtogether/readtogether/services/GrupoService.kt

```diff
@@ -20,4 +20,9 @@ class GrupoService @Autowired constructor(private val grupoRepository: GrupoRepo
     fun eliminarDatos(){
         grupoRepository.eliminarDatos()
     }
+
+    fun crearGrupo(nombre: String, detalle: String) {
+        val grupo1 = Grupo(nombre,detalle)
+        this.guardarGrupo(grupo1)
+    }
```

Comment:
Creeeeo que este metodo puede no estar, y directamente llamar a guardarGrupo pasandole grupos, entopnces esta creacion del grupo la hariamos en el controller

Reply:
no se a mi me parece que esta bien, lo dejo para futuro refactor

Reply:
:+1:

Reply:
```suggestion
    fun crearGrupo(nombre: String, detalle: String) {
        this.guardarGrupo(Grupo(nombre, detalle)
    }
```
sería igual que hacer esto, y ahi ya se te vuelve redundante el metodo